cmd/schema.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"time"
+
+	"github.com/cloudspannerecosystem/harbourbridge/common"
+	"github.com/cloudspannerecosystem/harbourbridge/conversion"
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	"github.com/google/subcommands"
+)
+
+// SchemaCmd struct with flags.
+type SchemaCmd struct {
+	filePrefix       string
+	driverName       string
+	schemaSampleSize int64
+	dumpFilePath     string
+	targetDb         string
+}
+
+// Name returns the name of operation.
+func (cmd *SchemaCmd) Name() string {
+	return "schema"
+}
+
+// Synopsis returns summary of operation.
+func (cmd *SchemaCmd) Synopsis() string {
+	return "generate schema for target db from source db schema"
+}
+
+// Usage returns usage info of the command.
+func (cmd *SchemaCmd) Usage() string {
+	return fmt.Sprintf(`%v schema [file]...
+
+Convert schema for source db specified by driver. Source db dump file can be
+specified by either dump-file flag or piped to stdin. Connection profile for source
+databases in direct connect mode can be specified by setting appropriate
+environment variables. The schema flags are:
+`, path.Base(os.Args[0]))
+}
+
+// SetFlags sets the flags.
+func (cmd *SchemaCmd) SetFlags(f *flag.FlagSet) {
+	f.StringVar(&cmd.filePrefix, "prefix", "", "File prefix for generated files")
+	f.StringVar(&cmd.driverName, "driver", "pg_dump", "Flag for specifying source DB or dump files (accepted values are \"pg_dump\", \"postgres\", \"mysqldump\", \"mysql\", and \"dynamodb\")")
```

Comment:
Just wondering out loud here, so feel free to ignore this.

Should we consider renaming driver to source?
We kind of have one chance to do flag renaming as part of the subcommand change, and as we add more capabilities to harbourbridge (including more target databases), I'm wondering if source and target are better/more-symmetric/more-futureproof that driver and targetDb?

Or should we adopt the terminology that DMS uses: source and destination?
(see https://cloud.google.com/database-migration/docs/overview).

Reply:
Yes, in general we want to move to terminology that DMS uses: source and destination as well as reuse the concept of connection profiles for connecting to source and destination databases. Since we only get once change at this, I wanted to be a little thorough in this renaming work and thus was planning to defer this as an immediate follow up work. WDYT?

Reply:
Yup, sounds good - doesn't need to be part of this PR.